samples/client/petstore/R/test_petstore.bash

```diff
@@ -4,7 +4,9 @@ set -e
 
 REPO=http://cran.revolutionanalytics.com
 
+export LC_ALL=en_US.UTF-8
 export R_LIBS_USER=$HOME/R
+export LANG=en_US.UTF-8
```

Comment:
@hpaknia why do you need these changes?

Reply:
@wing328 `ci/circleci` is failing. I'm trying to go around this error:
```
* checking package directory ... OK
* checking DESCRIPTION meta-information ... ERROR
npm WARN npm npm does not support Node.js v13.0.0
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11, 12.
npm WARN npm You can find the latest version at https://nodejs.org/
* DONE

Status: 1 ERROR
See
  ‘/home/circleci/OpenAPITools/openapi-generator/samples/client/petstore/R/petstore.Rcheck/00check.log’
for details.

[ERROR] Failed to execute goal or
```
Anyhow, the commit did not fix the issue. I dropped the second commit. Can you help me with debugging this https://circleci.com/gh/OpenAPITools/openapi-generator/9752?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link?

Reply:
The issue has fixed itself by CircleCI (likely they reverted a bad change in their side)

Latest is green: https://circleci.com/gh/OpenAPITools/openapi-generator/9767